components/util/constant.js

```diff
@@ -0,0 +1,24 @@
+export const ROUTE = {
+  랜딩: "/",
+  로그인: "/signin",
+  개인정보처리방침: "/privacy",
+  FAQ: "/faq",
+};
+
+const second = 1000;
+const minute = second * 60;
+const hour = minute * 60;
+const day = hour * 24;
+const month = day * 31;
+const year = month * 12;
```

Comment:
```suggestion
const SECOND = 1000;
const MINUTE = SECOND * 60;
const HOUR = MINUTE * 60;
const DAY = HOUR * 24;
const MONTH = DAY * 31;
const YEAR = MONTH * 12;
```

변하지 않는 상수는 UPPERCASE 컨벤션을 일반적으로 사용 합니다

그리고 객체로 묶지않고도 각각의 변수를 export 할 수 있어요

```typescript
export const SECOND = 1000;
export const MINUTE = SECOND * 60;
export const HOUR = MINUTE * 60;
export const DAY = HOUR * 24;
export const MONTH = DAY * 31;
export const YEAR = MONTH * 12;
```
앞에 export만 붙여주면 named export로 수출시킬 수 있습니다.

Reply:
UPPERCASE로 수정하겠습니다!